core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -429,40 +425,36 @@ public void listStatus() throws Exception {
    */
   @Test
   public void listStatusWithoutMountInfo() throws Exception {
-    FileInfo fileInfo1 = new FileInfo()
-        .setLastModificationTimeMs(111L)
-        .setLastAccessTimeMs(123L)
-        .setFolder(false)
-        .setOwner("user1")
-        .setGroup("group1")
-        .setMode(00755);
-    FileInfo fileInfo2 = new FileInfo()
-        .setLastModificationTimeMs(222L)
-        .setLastAccessTimeMs(234L)
-        .setFolder(true)
-        .setOwner("user2")
-        .setGroup("group2")
-        .setMode(00644);
-
```

Comment:
Could you tell readers a bit more about why you choose to remove the testing on `file infos`?